content/blog/2019/07/2019-07-03-remoting-kafka-kubernetes-phase-1.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Remoting over Apache Kafka plugin with Kafka launcher in Kubernetes"
+tags:
+- remoting
+- kafka
+- kubernetes
+- helm
+- developer
+- gsoc
+- gsoc2019
+author: longnguyen
+---
+
+= Remoting over Apache Kafka plugin with Kafka launcher in Kubernetes
+
+== Introduction
+
+Current version of Remoting over Apache Kafka plugin requires users to manually configure the entire system which includes Zookeeper, Kafka and remoting agents. It also doesn't support dynamic agent provisioning so scalability is harder to achieve. This project aims to solve two problems:
```

Comment:
"This project" does not help much those who do not follow GSoC. "My GSoC Project" with a project link would be better